odoo/addons/test_inherit/tests/test_inherit.py

```diff
@@ -174,3 +174,12 @@ def test_xml_ids(self):
             'test_inherit.selection__test_new_api_selection__state__baz',
         ])
 
+
+class TestInitColumn(common.TransactionCase):
+    def test_init_column(self):
+        # the installation of this module should:
+        #  - use the compute method to initialize field 'test_init_computed'
+        #  - bypass the compute method to initialize field 'test_init_optimized'
+        partner = self.env.user.partner_id
+        self.assertEqual(partner.test_init_computed, 'computed')
+        self.assertEqual(partner.test_init_optimized, 'optimized')
```

Comment:
This seems not related to inherit and if someone wants to find test related to field initialization they will not check ``test_inherit`` module. Maybe use ``test_new_api`` instead?

Reply:
It is related to inheritance in the way that inheritance adds columns to a non-empty table.
There was already an extension of model `res.partner` in that module, so this made the diff minimal for the test...